### ISSUE TITLE
Helm Package

### ISSUE THREAD

**Issue by kfox1111** [fluent/fluentd-kubernetes-daemonset#8]
It would be nice to get the daemonsets into helm at github.com/kubernetes/charts.

This means it would be as easy to install as:
helm install stable/fluentd-kubernetes-daemonset

**Comment by edsiper** [comment]
that is "work in process" :)

**Comment by kenden** [comment]
@edsiper Is there is place where we could check it? A PR maybe?

**Comment by edsiper** [comment]
@kenden this is the PR:

https://github.com/kubernetes/charts/pull/895

**Comment by kenden** [comment]
@edsiper  fluent-bit seems pretty cool. However I want to send the logs to Cloudwatch, and it doesn't seem to support it.
We would need to setup fluentd to receive the data from fluent-bit and send it to Cloudwatch. 
So we would still miss this bit: `helm install stable/fluentd-kubernetes-daemonset` ?

**Comment by edsiper** [comment]
No helm chart for Fluentd at the moment :/

**Comment by kenden** [comment]
WIP PR https://github.com/kubernetes/charts/pull/952, from https://github.com/kubernetes/charts/pull/211

**Comment by so0k** [comment]
did you manage to find out the cause of https://github.com/fluent/fluentd-kubernetes-daemonset/issues/13 ?

I used these images while testing logentries / logzio and loggly (we settled on loggly). 

I hope for PRs to fix ES and CloudWatch

**Comment by kenden** [comment]
@so0k No, it still appears in the logs. It's just a warning though, so it still works for most logs.

**Comment by kenden** [comment]
There is now a chart at:
https://github.com/kubernetes/charts/tree/master/incubator/fluentd-cloudwatch
So installation can now be done with 
```
helm install incubator/fluentd-cloudwatch
```

This ticket can be closed.

**Comment by kfox1111** [comment]
Any plan to support more then cloudwatch?

**Comment by kenden** [comment]
It would be nice to update that chart to support more.

For elasticsearch, it could reproduce what is there:
https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-elasticsearch

**Comment by monotek** [comment]
is there also a fluentd-elasticsearch chart available?

**Comment by kenden** [comment]
There is code for:
- logentries: https://github.com/kubernetes/charts/pull/779
- elasticsearch: https://github.com/kubernetes/kubernetes/tree/a1f20cbdf4bd2f29f1a9c19d8ca7dfa56ae22380/cluster/addons/fluentd-elasticsearch/fluentd-es-image

That could be incorporated to make the chart more generic. PRs welcome!

**Comment by monotek** [comment]
Unfortunately the Cloudwatch & fluent-bit images miss the detect exceptions plugin, which is really nice to have proper multiline logging for java exeptions and so on.

See: https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions

We're using it also outside of GCP.

**Comment by mvernimmen** [comment]
I've tried to follow the flow of different ideas and PR's to try and get a helm chart for fluentd that allows for k8s logs to be sent to a syslog destination. Almost a year ago this was 'work in progress' but it's not clear to me whether a solution currently exists.
https://github.com/kubernetes/charts/pull/779 was closed in preference of using https://github.com/kubernetes/charts/tree/master/incubator/fluentd-cloudwatch but it's unclear to me whether that can be used for shipping to a syslog destination. https://hub.kubeapps.com/charts/incubator/fluentd-cloudwatch Doesn't mention such capability.
Does someone have more insight on this please? Or an alternative? :)

**Comment by monotek** [comment]
You can now also try: https://github.com/kubernetes/charts/tree/master/incubator/fluentd-elasticsearch

**Comment by kfox1111** [comment]
oh great. I will try this along with the fluent-bit chart and the elasticsearch-operator chart. Should have all the pieces then to do shipping, processing, storage, and display then. :)

**Comment by timothyclarke** [comment]
I think this one is currently blocked by the read-only config map issue in #90

**Comment by bryan831** [comment]
anyone knows why 
`helm install incubator/fluentd-cloudwatch`  
is not working?
I tried and get the error:  
`Error: failed to download "incubator/fluentd-cloudwatch" (hint: running `helm repo update` may help)`  

I tried to run the suggested `helm repo update` but it didn't help.  
I am able to install other stable/* charts into my cluster..

**Comment by timothyclarke** [comment]
Is incubator in the list when you do `helm repo list` ?
Have you ever added incubator with `helm repo add` ?
